src/test/java/teammates/test/cases/ui/StudentCourseDetailsPageActionTest.java

```diff
@@ -17,6 +21,7 @@
 import teammates.ui.controller.ShowPageResult;
 import teammates.ui.controller.StudentCourseDetailsPageAction;
 import teammates.ui.controller.StudentCourseDetailsPageData;
+import teammates.ui.controller.StudentProfileEditSaveAction;
```

Comment:
Same mistake occurs: an irrelevant Action class being imported in the action test not related to it.